targets/TARGET_NUVOTON/TARGET_M251/CMakeLists.txt

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+if("NUMAKER_IOT_M252" IN_LIST MBED_TARGET_LABELS)
+    add_subdirectory(TARGET_NUMAKER_IOT_M252)    
+endif()
+
+if(${MBED_TOOLCHAIN} STREQUAL "GCC_ARM")
+    set(LINKER_FILE device/TOOLCHAIN_GCC_ARM/M251.ld)
+elseif(${MBED_TOOLCHAIN} STREQUAL "ARM")
+    set(LINKER_FILE device/TOOLCHAIN_ARM_STD/M251.sct)
+endif()
```

Comment:
Also support IAR linker file, such like as ?
```
elseif(${MBED_TOOLCHAIN} STREQUAL "IAR")
    set(LINKER_FILE device/TOOLCHAIN_IAR/M251.icf)
endif()
```

Reply:
we do not support IAR with CMake , thus we do not port it neither. If IAR is supported, we will need to add targets support (will be done via separate PRs).